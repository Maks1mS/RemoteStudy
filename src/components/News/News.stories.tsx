import News from './News'

export default {
  component: News,
  title: 'News'
}

export const text = (): JSX.Element => <News title="Hello world!" />
