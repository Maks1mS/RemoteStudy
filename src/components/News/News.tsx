import styled from '@emotion/styled'

const Container = styled.div`
  background: white;
  font-family: 'Roboto';
  border-radius: 5px;
`
type Props = {
  title: string
}

const News: React.FC<Props> = ({ title }) => {
  return <Container><h2>{title}</h2></Container>
}

export default News
