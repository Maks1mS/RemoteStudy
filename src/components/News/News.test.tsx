import { render } from '@testing-library/react'
import News from './News'

test('Renders', () => {
  const { container } = render(<News title="Test"/>)
  expect(container).toMatchSnapshot()
})
