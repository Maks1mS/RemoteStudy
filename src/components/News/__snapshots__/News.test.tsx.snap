// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div>
  <div
    class="css-1u4rf2v-Container e1dz631m0"
  >
    <h2>
      Test
    </h2>
  </div>
</div>
`;
